src/app/frontend/_rtl.scss

```diff
@@ -0,0 +1,68 @@
+// Copyright 2017 The Kubernetes Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+body:lang(fa) {
+  direction: rtl;
+
+  .mat-form-field {
+    text-align: unset;
+  }
+
+  .mat-paginator-range-actions {
+    flex-direction: row-reverse;
+  }
+
+  .kd-slider-value {
+    text-align: left;
+  }
+
+  .mat-option {
+    text-align: right;
+  }
+
+  .mat-optgroup {
+    .mat-option {
+      padding-left: 16px;
```

Comment:
Import `variables.scss` and use baseline-grid as in other files.

Reply:
> Import `variables.scss` and use baseline-grid as in other files.

Yes, you are right. I fixed it.